src/rules/typescript.ts

```diff
@@ -8,6 +8,8 @@ import memberOrderingList from './rule-helpers/member-ordering-list';
 export default {
   // Enforces function overloads to be consecutive.
   'adjacent-overload-signatures': true,
+  // Bans specific types from being used. Does not ban the corresponding runtime objects from being used. <FEEDBACK>
+  'ban-types': true,
```

Comment:
Thoughts on enabling this rule. 
With this we can ban specific Types from being used, any come to mind?
[See here for example test cases](https://github.com/palantir/tslint/tree/master/test/rules/ban-types)